kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -507,3 +507,22 @@ def image_for_product(product_slug):
     except Product.DoesNotExist:
         return os.path.join(settings.STATIC_URL, 'products', 'img', 'product_placeholder.png')
     return obj.image_url
+
+
+@jinja2.contextfunction
+@library.global_function
+def show_fx_download(context):
```

Comment:
Looks like you decided against using this?

Reply:
It's used in `sumo/jinja2/base.html` - only for the download button in the header.